logger.ts

```diff
@@ -0,0 +1,23 @@
+const winston = require("winston");
+const { combine, timestamp, json, errors } = winston.format;
+const { Logtail } = require("@logtail/node");
+const { LogtailTransport } = require("@logtail/winston");
+
+const logtail = new Logtail("fLhHrFNbdBe9DFFqSyp9RZZk");
```

Comment:
Is this Logtail apiKey?
If yes please move it to the env file. You should never expose api keys to the public.

Reply:
Damn!! Yes, it is.

Should I delete this pull request and create a new one, so API key doesn't appear anywhere in the GitHub history?

Reply:
You don't have to, just add it to the .env file. On vercel create one additional environment variable for logtail so that it works on production as well.

Reply:
I've moved the key to the .env file and added it to environment variables on Vercel project.